cli/lib/validation.ts

```diff
@@ -1,10 +1,13 @@
 import { Quad } from 'rdf-js'
+import $rdf from 'rdf-ext'
 import type { Context } from 'barnard59-core/lib/Pipeline'
 import { validateQuad } from 'rdf-validate-datatype'
 
 export function validate(this: Context, quad: Quad): Quad {
   if (!validateQuad(quad)) {
-    throw new Error(`Quad has invalid datatype: ${quad}`)
+    // Take advantage of rdf-ext's `toString`
+    const quadExt = $rdf.quad(quad.subject, quad.predicate, quad.object)
+    throw new Error(`Quad object has invalid datatype:\n\t${quadExt.toString()}`)
```

Comment:
You could try with `rdf-string`, gaining automatic term prefixing: https://runkit.com/embed/uizz61o7twqr

Reply:
Will do, thanks

Reply:
Ah, because named graph: https://runkit.com/embed/2csifrphpmw5